examples/benchmark-ui/elements.py

```diff
@@ -0,0 +1,102 @@
+
+class TextElements:
+    
+    # Markdown Text
+    md_title = "# Benchmarking Sparse Models on DigitalOcean's CPUs!"
+
+    md_body = '''
+
+    ## Hey There 👋
```

Comment:
maybe regular text size? there already is a title header that describes it

Reply:
can we lower the heading size to "### " instead of normal text size? Keeping it regular text size looks a bit weird aesthetically. 🤷‍♂️